polaris-ui/src/app/features/cases/presentation/case-details/modals/SaveUsedOrUnusedModal.tsx

```diff
@@ -0,0 +1,77 @@
+import { useState, useEffect } from "react";
+import {
+  LinkButton,
+  Modal,
+  Spinner,
+} from "../../../../../common/presentation/components/index";
+import { ReactComponent as WhiteTickIcon } from "../../../../../common/presentation/svgs/whiteTick.svg";
+import { ReactComponent as CloseIcon } from "../../../../../common/presentation/svgs/closeIconBold.svg";
+import classes from "./SaveUsedOrUnusedModal.module.scss";
+
+type Props = {
+  savingState?: string;
+};
+export const SaveUsedOrUnusedModal: React.FC<Props> = ({ savingState }) => {
+  const [showModal, setShowModal] = useState(false);
+
+  useEffect(() => {
+    if (savingState === "saving") setShowModal(true);
+  }, [savingState]);
+
+  const handleCloseModal = () => {
+    setShowModal(false);
+  };
+
+  return (
+    <>
```

Comment:
you might need to add aria-live for letting screen reader know about saving to cms text , please refer to SaveRotationModla.tsx.    ```
<div>
      <div aria-live="polite" className={classes.visuallyHidden}>
        {saveStatus.type === "rotation" && saveStatus.status === "saving" && (
          <span>Saving updated document to CMS</span>
        )}
      </div>
```